sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveDDLSuite.scala

```diff
@@ -391,6 +391,29 @@ class HiveDDLSuite
     }
   }
 
+  test("create view with mismatched schema") {
```

Comment:
Did you copy the tests? Could you file a JIRA issue to get rid of the duplication at the very least? Thanks.